src/components/list/CategoryMenu.tsx

```
@@ -56,7 +56,11 @@ export default function CategoryMenu({ categoryId, handleCategoryError }: Catego
     return (
         <div className="w-full mx-0 py-4" data-cy="category-list">
         {isLoading ?
-            <Spinner message="카테고리를 불러오는 중입니다..." />
+            <div className="flex gap-4">
+                {[...Array(3)].map((_, i) => (
+                    <div key={i} className={`w-20 h-12 rounded-lg bg-gray-${3 - i}00 animate-pulse`} />
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)